test-cases/performance/perf-regression-latency-k8s-multitenant.yaml

```diff
@@ -0,0 +1,30 @@
+# NOTE: need to define following config options in the 'configurations' files:
+# - prepare_write_cmd
+# - stress_cmd_w
+# - stress_cmd_m
+# - stress_cmd_r
+# - n_loaders
+# - k8s_n_loader_pods_per_cluster
+
+test_duration: 480
+user_prefix: 'perf-regression-latency'
```

Comment:
better to add the `k8s` to this user prefix

Reply:
Why better? It is used in K8S cluster names. All the K8S VMs get auto-scheduled by node groups and have no names at all.

Reply:
to have a clear separation to other `'perf-regression-latency'` tests, in case there will be resources/emails/logs that could eventually cause doubt on which `'perf-regression-latency'` it is related to